packages/components/.babelrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  presets: [
+    "@babel/preset-typescript",
+    "@babel/preset-react",
+    [
+      "@babel/preset-env",
+      {
+        targets: [
+          "last 2 chrome versions",
+          "last 2 firefox versions",
+          "last 2 safari versions",
+        ],
```

Comment:
tsc wasn't previously doing any crazy transformations, or polyfilling anything. So targeting recent versions here feels pretty safe.

We can always tweak this as necessary.